src/js/components/SelectMultiple/__tests__/__snapshots__/SelectMultiple-test.tsx.snap

```diff
@@ -4973,58 +4973,6 @@ exports[`SelectMultiple showSelectionInline 1`] = `
   border: 0;
 }
 
-.c19 {
```

Comment:
Any idea what happened here? Shouldn't be affected?

Reply:
:( no I need to look into this